pkg/migrations/dbactions.go

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package migrations
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/lib/pq"
+	"github.com/xataio/pgroll/pkg/db"
+)
+
+// DBAction is an interface for common database actions
+// pgroll runs during migrations.
+type DBAction interface {
+	Execute(context.Context) error
+}
```

Comment:
We should consider moving this interface and implementations into a separate `dbactions` package IMO.

Reply:
I agree, I will move it in a future PR. My initial goal with this PR was to get your feedback on the idea.